packages/expo-cli/src/commands/eas-build/build/action.ts

```diff
@@ -116,12 +130,16 @@ async function startBuildsAsync(
 async function startBuildAsync(
   client: ApiV2,
   builder: Builder,
-  projectId: string
+  projectId: string,
+  skipProjectConfiguration: boolean
 ): Promise<string> {
   const tarPath = path.join(os.tmpdir(), `${uuidv4()}.tar.gz`);
   try {
     await builder.ensureCredentialsAsync();
-    await builder.configureProjectAsync();
+
+    if (!skipProjectConfiguration) {
```

Comment:
```suggestion
    if (!builder.ctx.skipProjectConfiguration) {
```
You don't need to pass that as argument